lib/OpenQA/WebAPI.pm

```diff
@@ -54,9 +54,11 @@ sub startup {
     my $r         = $self->routes;
     my $logged_in = $r->under('/')->to("session#ensure_user");
     my $auth      = $r->under('/')->to("session#ensure_operator");
+    my $plugin_r  = $auth->route('/plugin')->to(namespace => 'WebAPIPlugin');
 
     OpenQA::Setup::setup_template_search_path($self);
     OpenQA::Setup::load_plugins($self, $auth);
+    OpenQA::Setup::load_ui_plugins($self, $plugin_r);
```

Comment:
Would these plugins also be loaded within the scheduler, liveview handler and websocket server? Not sure whether this is a good idea.

Reply:
They just don't have to call `OpenQA::Setup::load_ui_plugins()` as far as i understand this PR.

Reply:
Right, this call is within WebAPI.pm. So everything is good.